etools-loading-behavior.html

```diff
@@ -1,9 +1,18 @@
 <link rel="import" href="etools-loading.html">
+<link rel="import" href="scripts/lodash.html">
```

Comment:
the problem is when the project and the component use different versions of lodash. It will need to be tested to see how it behaves, especially in IE